src/information-panel/information-panel.scss

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-information-panel-wrapper {
+  position: relative;
+  overflow: hidden;
+}
+
+@mixin iui-information-panel {
+  position: absolute;
+  opacity: 0;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  z-index: 2; // Needs to be higher than table's column resizers.
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+
+  > .iui-resizer {
+    display: none;
+    position: absolute;
+    touch-action: none;
+    z-index: 1000;
+
+    > .iui-resizer-bar {
+      @include themed {
+        background-color: t(iui-color-background-5);
+      }
+    }
+
+    &:hover > .iui-resizer-bar {
+      @include themed {
+        background-color: t(iui-color-foreground-primary);
+      }
+    }
+  }
+
+  &.iui-information-panel-visible {
+    opacity: 1;
+
+    > .iui-resizer {
+      display: flex;
+    }
+  }
+
+  .iui-information-header,
+  .iui-information-body {
+    padding-left: $iui-sm;
+    padding-right: $iui-sm;
+    box-sizing: border-box;
+  }
+
+  .iui-information-header {
+    height: $iui-baseline * 6;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    @include themed {
+      background-color: t(iui-color-background-3);
+    }
+
+    .iui-header-label {
```

Comment:
we need to rename this
`iui-information-header-label`

Reply:
Done. We should copy this naming format in #264 as well.